pkg/visor/rpc.go

```diff
@@ -390,3 +393,16 @@ func (r *RPC) Loops(_ *struct{}, out *[]LoopInfo) error {
 	*out = loops
 	return nil
 }
+
+/*
+	<<< VISOR MANAGEMENT >>>
+*/
+
+// Restart restarts visor.
+func (r *RPC) Restart(_ *struct{}, _ *struct{}) error {
+	if r.node.restartCtx == nil {
```

Comment:
Should we protect this with a `sync.Mutex`? As it states in the `(*restart.Context).Restart()` method comment: 
```golang
// Should not be called from a goroutine.
```

Reply:
@evanlinjin You're right, thanks.